Helper/Image.php

```diff
@@ -10,8 +10,11 @@
 
 class Image extends \Magento\Catalog\Helper\Image
 {
+    /**
+     * @var ConfigHelper
+     */
+    public $configHelper;
```

Comment:
Same thing here :) 
Thanks in advance.

Reply:
I'd rather not. Shall we compromise and make them protected? Private leaves me with very few nice options to override these.

Reply:
Let's go for protected :)